components/post_view/post_body_additional_content/post_body_additional_content.tsx

```diff
@@ -113,7 +113,15 @@ export default class PostBodyAdditionalContent extends React.PureComponent<Props
                     toggleEmbedVisibility={this.toggleEmbedVisibility}
                 />
             );
-
+        case 'permalink':
+            if (embed.data && 'post_id' in embed.data) {
```

Comment:
`'post_id' in embed.data` will be `true` even if the value is `null`.

Reply:
This is to check that embed.data is actually using the type: `PostPreviewMetadata`. `data` can also be of the type, `OpenGraphMetadata`. 

I can also add a check for `null` or `undefined` though